scripts/build.py

```diff
@@ -142,50 +257,49 @@ def build_files(source, target):
                 ensure_directory_exists(target_path)
                 shutil.copyfile(source_path, target_path)
 
-def get_cache_slug():
-    """Returns the cache slug read from file."""
-    with open('cache_slug.yaml', 'r') as cache_slug_file:
-        content = cache_slug_file.read()
-    retrieved_dict = yaml.safe_load(content)
-    assert isinstance(retrieved_dict, dict)
-    return retrieved_dict['cache_slug']
-
-
 if __name__ == '__main__':
-    CACHE_SLUG = get_cache_slug()
-    BUILD_DIR = os.path.join('build', CACHE_SLUG)
-
-    # os.path.dirname(path)(in ensure_directory_exists()) returns parent
-    # directory of a path passed as an argument to it. This is as intended
-    # for file paths, but for directory paths we do not want this to happen,
-    # hence we append a trailing slash to it.
-    # Process assets, copy it to build/[cache_slug]/assets
-    ASSETS_SRC_DIR = os.path.join('assets', '')
-    ASSETS_OUT_DIR = os.path.join(BUILD_DIR, 'assets', '')
-    copy_files_source_to_target(ASSETS_SRC_DIR, ASSETS_OUT_DIR)
+    hashes = dict() # pylint: disable=C0103
```

Comment:
Ah, I think I know. It might be because it's in global scope. To get around this, declare a new function and have the bottom of the file just be something like

```
    if __name__ == '__main__':
        build_files()   <-- pick a suitable name here based on whatever the function does
```

I think that might fix it.